packages/app/src/cli/models/extensions/extension-instance.ts

```diff
@@ -227,6 +229,15 @@ export class ExtensionInstance<TConfiguration extends BaseConfigType = BaseConfi
     return this.specification.hasExtensionPointTarget?.(this.configuration, target) || false
   }
 
+  getPrefixedLogger(wrappedWritable: Writable) {
+    const handle = this.handle
+    return new Writable({
+      write(chunk, _encoding, callback) {
+        wrappedWritable.write(`${prefixConcurrentOutputLog(handle, chunk.toString('utf8'))}`, 'utf8', callback)
+      },
+    })
+  }
```

Comment:
So we are creating multiple Writable objects, but in reality we are writing to the same one and using this as a "proxy". Does this has any downsides?